test/test_nestedtensor.py

```diff
@@ -3555,9 +3614,11 @@ def test_noncontiguous_pointwise(self, device):
         nt, _ = jagged_from_list([a, b, c], None)
         # transpose ragged dim
         transposed = nt.transpose(1, 2)
-        # pointwise ops are not supported on ragged dim transposed jagged layout NTs
-        with self.assertRaisesRegex(ValueError, "expected .* to be a contiguous jagged layout"):
-            clone = transposed.clone()
+        clone = transposed.clone()
```

Comment:
please assert on contiguity here - we expect the clone to be contiguous but transposed not to be

Reply:
I checked in eager non-NT; it seems like contiguity isn't actually preserved usually? I added tests for this, and also asserts that is_contiguity is true after `nt.transpose(1, 2).clone(memory_format=torch.contiguous_format)`

Reply:
> I checked in eager non-NT; it seems like contiguity isn't actually preserved usually

that's right - my understanding is that regardless of input contiguity, `clone()` will spit out a contiguous tensor

Reply:
> regardless of input contiguity, clone() will spit out a contiguous tensor

this was not what I observed (both with NT and eager) - usually it spits out a tensor with the same strides as the input, unless `contiguous_format` is specified

Reply:
ah I see, thanks for clarifying!